docs/source/home.rst

```diff
@@ -24,6 +24,7 @@ you will always be able to interact with your environment, engage with your frie
 
 
 In addition to the documentation, you can join the discussion on the Vircadia `Discord server <https://discordapp.com/invite/Pvx2vke>`__ |Discord|.
+To join the developer discussions ask a moderator to be included.
```

Comment:
This definitely doesn't belong on the home page

Reply:
so move this discord invite off the front page as well?

Reply:
Currently Discord is the main mean for new users to ask questions, so it should definitely be included here. Obviously you can also join developer discussion if you are not in the more private channel, which is also the main reason why I would want to keep this on the front page. A new user that is potentially interested in development can ask questions and hopefully be involved in an inspiring conversation with another user or developer.

For the contributors-only channel is interesting to keep the traffic low (contributors being able to prioritize their "collegues" when they are too busy to keep up with the whole discord server), and talking about things that one might want to keep semi private, like talking about internal topics that have led to drama before without involving "outsiders".

Reply:
Not sure what the pushback is exactly? do you knows not want people to know about communication options?

Reply:
The pushback is that information about a contributor channel needs to be in the right place. The front page needs to stay an overview rather than going into detail like that, or it will become too big and people won't read or even skimm over it.

Personally I am also of the opinion that the existence of a contributor-only channel shouldn't matter to non-contributors. But that is different from the issue here.